corporate/views/remote_billing_page.py

```diff
@@ -0,0 +1,175 @@
+import logging
+from typing import Optional, TypedDict
+
+from django.conf import settings
+from django.core import signing
+from django.http import HttpRequest, HttpResponse, HttpResponseRedirect
+from django.shortcuts import render
+from django.urls import reverse
+from django.utils.translation import gettext as _
+from django.views.decorators.csrf import csrf_exempt
+from pydantic import Json
+
+from zerver.lib.exceptions import JsonableError, MissingRemoteRealmError
+from zerver.lib.remote_server import RealmDataForAnalytics
+from zerver.lib.response import json_success
+from zerver.lib.typed_endpoint import PathOnly, typed_endpoint
+from zilencer.models import RemoteRealm, RemoteZulipServer
+
+billing_logger = logging.getLogger("corporate.stripe")
+
+
+SIGNED_ACCESS_TOKEN_VALIDITY_IN_SECONDS = 60 * 60 * 24
```

Comment:
Is there a reason for this to be more than 30 seconds? As best I can tell, the way we use this is such that you're immediately redirected to a page that will use this and store the values in the session, so this is intended to be a very temporary URL construction.

Reply:
I guess in the file upload code path, we have a similar value that's 60s, so maybe we want to use that value but also move to it to a `zproject/default_settings.py` value used in both places; the name `SIGNED_ACCESS_TOKEN_VALIDITY_IN_SECONDS` seems good for it.

`zerver/views/upload.py:        path_id = signer.unsign(signed_data, max_age=timedelta(seconds=60))`